src/containers/__tests__/AccountPanel.test.tsx

```diff
@@ -0,0 +1,68 @@
+import { AccountPanelComponent } from '@containers/AccountPanel'
+import { NotificationType } from '@types'
+import { shallow } from 'enzyme'
+import { Keypair } from 'js-kinesis-sdk'
+import * as React from 'react'
+import '../../setupTests'
+
+describe.only('AccountPanel', () => {
+  let props
+
+  beforeEach(() => {
+    props = {
```

Comment:
This means you don't get any type errors for props being passed in this file. Unsure if it's intended. Don't think it's a great idea and can save time if just defined as a const above (as you ran into issues with it).